README.md

```diff
@@ -83,6 +83,23 @@ Currently there is an issue where the above leads to react complaining about mul
 4. Open terminal in `glints-aries`.
 5. Run `yarn link react`.
 
+### Adding visual test
+
+We use [playwright](https://playwright.dev/docs/test-snapshots) visual comparison test for components to check for regressions, since images in every browser and machine can be rendered differently we'll use docker image to generate and compare the screenshots.
+
+All visual test is in `test/e2e` directory
+
+To generate new screenshots locally:
```

Comment:
maybe we can mention where the screenshots can be located?

Reply:
updated